scripts/praxis_development.sh

```diff
@@ -0,0 +1,37 @@
+# Please make sure that you have your device selected in the Flutter Device Section before running this script
+
+# Go to root dir from the current scripts dir
+echo "<---Going to root directory--->"
+cd ..
+
+# Use pub get to fix pubspec.lock error
+flutter pub get
+
+# Generate config files for the presentation layer
+echo "<---Generating config files for the presentation layer--->"
+flutter packages pub run build_runner build --delete-conflicting-outputs
+
+# Go to data layer (package)
+echo "<---Going to data layer--->"
+cd praxis_data || return
```

Comment:
@shubhamsinghmutualmobile we need to first generate for praxis_data and then root.

Reply:
Okay

Reply:
Done